[watch-js] Adiciona watch nos arquivos JS (#62)
elo7/doc-amd: README.md

});
```

### Script Commands

| | |

> |-|-|

Reviewer: Precisa dessas duas linhas? Parece que criou uma linha vazia na tabela final...

Author: Na teoria precisa sim para poder criar tabela no markdown. =/ 
Mas podemos fazer sem tabela mesmo, e suar apenas lista não ordenada! O que acha? :thinking:

Reviewer: Acho que tabela ainda é a melhor opção, é a mais semântica a meu ver. Pelo que entendi olhando a documentação do Markdown, a primeira linha é o cabeçalho da tabela. Sendo assim, que tal preenchê-la com texto? Tipo:
```suggestion
|Script|Description|
|-|-|
```

Author: Justo! 
:hammer_and_pick:

| `npm start` | Install all dependencies and start the server. |

> | `npm run start:dev` | Run all it exists at the start script and run the watch script. |

Reviewer: ```suggestion
| `npm run start:dev` | Run the start script and then the watch script. |
```

Author: :hammer_and_pick:

> | `npm run test` | Run test local. |

Reviewer: ```suggestion
| `npm run test` | Run test locally. |
```

Author: :hammer_and_pick:

> | `npm run test:ci` | Run test with together server for to continuous integration. |

Reviewer: ```suggestion
| `npm run test:ci` | Run the server and the tests. Useful for a quick local check and for CI environment.  |
```

Author: Desculpe mas é que o meu inglês não é muito bom :cry: 

:hammer_and_pick:
